"Id","ParentId","Type","CreatedById","CreatedDate","Body","Title","LinkUrl","RelatedRecordId","InsertedById","LastModifiedDate"
"0D5Qk00000UrHXoKAN","500Qk00000KdHVJIA3","TextPost","0054u000008ARZjAAO","2025-01-15T06:36:52.000Z","Description: Transaction (Process ID 86) was deadlocked on lock resources with another process and has been chosen as the deadlock victim. Rerun the transaction. The collection job returned error and subsequent reports generated were not having correct data.","","","","0054u000008ARZjAAO","2025-01-15T06:36:52.000Z"
"0D5Qk00000UshVdKAJ","500Qk00000KdHVJIA3","TextPost","005Qk000002b4gDIAQ","2025-01-15T15:57:54.000Z","What We've Learned Updated: Customer has a customer ""SHELL_ALLUserAccount"" activity scan job that is failing with this error:
""Transaction (Process ID 86) was deadlocked on lock resources with another process and has been chosen as the deadlock victim. Rerun the transaction. The collection job returned error and subsequent reports generated were not having correct data.""","","","","005Qk000002b4gDIAQ","2025-01-15T15:57:54.000Z"
"0D5Qk00000Usl2cKAB","500Qk00000KdHVJIA3","TextPost","005Qk000002b4gDIAQ","2025-01-15T15:58:24.000Z","Next Steps Updated: Support asked customer to provide additional details and collect debug logs.","","","","005Qk000002b4gDIAQ","2025-01-15T15:58:24.000Z"
"0D5Qk00000UwZXWKA3","500Qk00000KdHVJIA3","TextPost","005Qk000002b4gDIAQ","2025-01-16T12:26:31.000Z","What We've Learned Updated: Customer has a customer ""SHELL_ALLUserAccount"" activity scan job that is failing with this error:
""Transaction (Process ID 86) was deadlocked on lock resources with another process and has been chosen as the deadlock victim. Rerun the transaction. The collection job returned error and subsequent reports generated were not having correct data.""

Customer replied that the error is resolved on its own and activity scan is now running as expected.","","","","005Qk000002b4gDIAQ","2025-01-16T12:26:31.000Z"
"0D5Qk00000UwcTjKAJ","500Qk00000KdHVJIA3","TextPost","005Qk000002b4gDIAQ","2025-01-16T12:27:25.000Z","Next Steps Updated: Since error got resolved on its own and activity scan ""SHELL_ALLUserAccount"" is now running as expected support is waiting for a reply from customer with results. 
Customer will upload logs if issue is still present.","","","","005Qk000002b4gDIAQ","2025-01-16T12:27:25.000Z"
